app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,24 @@
+require "./lib/auth0_api"
+
+class UserMailer < ApplicationMailer
+  def welcome(user_id)
```

Comment:
Any reason you're taking a `user_id` here instead of just sending in the `user` instance directly, as per the [Action Mailer docs](https://guides.rubyonrails.org/action_mailer_basics.html#calling-the-mailer) ?

Reply:
I was working on the old understanding that you couldn't send objects into Redis via Sidekiq. Can we do this given that Sidekiq is the Action Mailer queue here?

Reply:
It'll use Active job and that'll handle the serialisation before it get sent to Sidekiq.